docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -390,6 +390,7 @@ relation to port exhaustion.
    authentication.
 - `REVERSE_PROXY_AUTHENTICATION_EMAIL`: **X-WEBAUTH-EMAIL**: Header name for reverse proxy
    authentication provided email.
+- `REVERSE_PROXY_FORWARDED_IP`: **false**: Interpret X-Forwarded-For header or the X-Real-IP header and set this as the remote IP for the request
```

Comment:
Rather than a true/false, should we instead have an array of IP ranges that we can accept reverse proxy headers from?